tests/unit/forklift/test_legacy.py

```diff
@@ -1396,7 +1396,10 @@ def test_upload_fails_with_too_large_file(self, pyramid_config,
         resp = excinfo.value
 
         assert resp.status_code == 400
-        assert resp.status == "400 File too large."
+        expected_msg = "400 File too large. Limit for project {!r} is {}MB"
+        expected_msg = expected_msg.format(project.name,
+                                           legacy.MAX_FILESIZE // (1024**2))
+        assert resp.status == expected_msg
```

Comment:
To make this test a little easier to read, we could explicitly set a name and upload limit when creating the project being tested:

```py
project = ProjectFactory.create(
    name="foobar",
    upload_limit=upload_limit=60 * 1024 * 1024  # 60MB
)
```

We can use that upload limit instead of `legacy.MAX_FILESIZE` when creating the test file:
```py
file=io.BytesIO(b"a" * (project.upload_limit + 1)),
```

And then it would be OK to make this assertion instead:
```py
assert resp.status == "400 File too large. Limit for project 'foobar' is 60MB"
```

Which is nice because we can just look at the test and know it's what we want.